bundle/manifests/odf-multicluster-orchestrator.clusterserviceversion.yaml

```diff
@@ -278,9 +278,7 @@ spec:
                 control-plane: odfmo-controller-manager
             spec:
               containers:
-              - args:
-                - --ssl --cert=/var/serving-cert/tls.crt --key=/var/serving-cert/tls.key
```

Comment:
switched to Nginx to run odf-multicluster-console, and it will takecare of passing these certificates: https://github.com/red-hat-storage/odf-console/blob/master/default.conf